Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642817): Gdk-CRITICAL **: 18:14:39.716: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641451): Gdk-CRITICAL **: 18:14:39.860: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641676): Gdk-CRITICAL **: 18:14:39.921: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641800): Gdk-CRITICAL **: 18:14:39.982: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640731): Gdk-CRITICAL **: 18:14:39.989: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641002): Gdk-CRITICAL **: 18:14:40.039: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641727): Gdk-CRITICAL **: 18:14:40.056: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640696): Gdk-CRITICAL **: 18:14:40.071: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641047): Gdk-CRITICAL **: 18:14:40.107: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640857): Gdk-CRITICAL **: 18:14:40.126: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642831): Gdk-CRITICAL **: 18:14:40.139: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640922): Gdk-CRITICAL **: 18:14:40.146: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640710): Gdk-CRITICAL **: 18:14:40.184: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642773): Gdk-CRITICAL **: 18:14:40.216: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641474): Gdk-CRITICAL **: 18:14:40.250: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640975): Gdk-CRITICAL **: 18:14:40.256: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640802): Gdk-CRITICAL **: 18:14:40.307: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed

(DataCollector_Deterministic.py:1641281): Gdk-CRITICAL **: 18:14:40.316: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641063): Gdk-CRITICAL **: 18:14:40.348: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed

(DataCollector_Deterministic.py:1641546): Gdk-CRITICAL **: 18:14:40.357: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641971): Gdk-CRITICAL **: 18:14:40.394: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641937): Gdk-CRITICAL **: 18:14:40.412: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642783): Gdk-CRITICAL **: 18:14:40.415: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641252): Gdk-CRITICAL **: 18:14:40.433: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641107): Gdk-CRITICAL **: 18:14:40.440: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640716): Gdk-CRITICAL **: 18:14:40.449: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642293): Gdk-CRITICAL **: 18:14:40.485: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641154): Gdk-CRITICAL **: 18:14:40.557: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641231): Gdk-CRITICAL **: 18:14:40.598: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641013): Gdk-CRITICAL **: 18:14:40.601: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642430): Gdk-CRITICAL **: 18:14:40.854: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642522): Gdk-CRITICAL **: 18:14:40.886: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640690): Gdk-CRITICAL **: 18:14:41.053: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640702): Gdk-CRITICAL **: 18:14:41.137: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641344): Gdk-CRITICAL **: 18:14:45.878: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642445): Gdk-CRITICAL **: 18:14:49.407: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641699): Gdk-CRITICAL **: 18:14:49.412: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642325): Gdk-CRITICAL **: 18:14:49.469: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641209): Gdk-CRITICAL **: 18:14:49.516: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640662): Gdk-CRITICAL **: 18:14:49.874: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642794): Gdk-CRITICAL **: 18:14:52.335: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641549): Gdk-CRITICAL **: 18:14:52.841: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640949): Gdk-CRITICAL **: 18:14:52.848: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640899): Gdk-CRITICAL **: 18:14:52.851: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642140): Gdk-CRITICAL **: 18:14:52.865: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641810): Gdk-CRITICAL **: 18:14:52.901: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642176): Gdk-CRITICAL **: 18:14:53.069: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641995): Gdk-CRITICAL **: 18:14:53.081: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640683): Gdk-CRITICAL **: 18:14:54.110: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642080): Gdk-CRITICAL **: 18:14:54.826: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed

(DataCollector_Deterministic.py:1641030): Gdk-CRITICAL **: 18:14:54.837: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642435): Gdk-CRITICAL **: 18:14:55.185: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642309): Gdk-CRITICAL **: 18:14:55.233: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642463): Gdk-CRITICAL **: 18:14:55.261: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642568): Gdk-CRITICAL **: 18:14:55.274: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642558): Gdk-CRITICAL **: 18:14:55.283: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641743): Gdk-CRITICAL **: 18:14:55.299: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642695): Gdk-CRITICAL **: 18:14:55.305: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642124): Gdk-CRITICAL **: 18:14:55.314: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641301): Gdk-CRITICAL **: 18:14:55.338: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642621): Gdk-CRITICAL **: 18:14:55.341: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642097): Gdk-CRITICAL **: 18:14:55.355: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640812): Gdk-CRITICAL **: 18:14:55.364: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642856): Gdk-CRITICAL **: 18:14:55.366: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641481): Gdk-CRITICAL **: 18:14:55.369: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641593): Gdk-CRITICAL **: 18:14:55.378: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642809): Gdk-CRITICAL **: 18:14:55.386: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642395): Gdk-CRITICAL **: 18:14:55.389: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642756): Gdk-CRITICAL **: 18:14:55.397: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642826): Gdk-CRITICAL **: 18:14:55.424: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642802): Gdk-CRITICAL **: 18:14:55.426: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642066): Gdk-CRITICAL **: 18:14:55.433: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642146): Gdk-CRITICAL **: 18:14:55.449: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641618): Gdk-CRITICAL **: 18:14:55.458: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641955): Gdk-CRITICAL **: 18:14:55.495: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641464): Gdk-CRITICAL **: 18:14:55.506: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641918): Gdk-CRITICAL **: 18:14:55.518: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641410): Gdk-CRITICAL **: 18:14:55.521: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641072): Gdk-CRITICAL **: 18:14:55.535: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641794): Gdk-CRITICAL **: 18:14:55.543: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642345): Gdk-CRITICAL **: 18:14:55.558: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641395): Gdk-CRITICAL **: 18:14:55.567: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642590): Gdk-CRITICAL **: 18:14:55.570: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641146): Gdk-CRITICAL **: 18:14:55.592: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641709): Gdk-CRITICAL **: 18:14:55.606: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642675): Gdk-CRITICAL **: 18:14:55.615: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641421): Gdk-CRITICAL **: 18:14:55.619: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642050): Gdk-CRITICAL **: 18:14:55.682: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642735): Gdk-CRITICAL **: 18:14:55.719: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed

(DataCollector_Deterministic.py:1642624): Gdk-CRITICAL **: 18:14:55.728: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641492): Gdk-CRITICAL **: 18:14:55.748: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641383): Gdk-CRITICAL **: 18:14:55.774: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642014): Gdk-CRITICAL **: 18:14:55.847: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640744): Gdk-CRITICAL **: 18:14:55.883: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641905): Gdk-CRITICAL **: 18:14:55.907: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642665): Gdk-CRITICAL **: 18:14:55.913: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642104): Gdk-CRITICAL **: 18:14:55.949: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642190): Gdk-CRITICAL **: 18:14:55.979: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641650): Gdk-CRITICAL **: 18:14:56.059: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642368): Gdk-CRITICAL **: 18:14:56.066: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
...........................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................
[1689 9145]
.....
[1758 9140]
.
[9153 1759]
...
[9133 1753]
.
[1744 9147]
.
[1730 9110]
.
[1713 9139]
.
[1709 9171]
.
[1690 9157]
..
[1730 9129]
.....
[1742 9093]
.
[1685 9171]
.
[1688 9142]
.
[1688 9181]
.
[1676 9181]
.
[1662 9221]
.
[1713 9146]
...
[1766 9096]
.
[1859 9047]
.
[1756 9102]
...
[1765 9122]
.
[1690 9147]
..
[1691 9140]
.
[1768 9133]
..
[1715 9169]
.
[1726 9140]
.
[1726 9073]
.
[1771 9067]
.
[1778 9063]
.
[1756 9143]
.
[1746 9084]
...
[1804 9115]
.
[1809 9070]
..
[1736 9110]
.
[1687 9151]
.
[1757 9125]
.
[1675 9148]
.
[1761 9070]
.
[1761 9106]
.
[1684 9125]
..
[1746 9046]
.
[2251 8547]
.
[1792 9056]
.
[1747 9085]
.
[1745 9133]
.
[1774 9078]
.
[1716 9065]
.
[1808 9078]
.
[1815 9067]
.
[1890 8959]
.
[1747 9061]
..
[1779 9100]
.
[1742 9089]
.
[1732 9078]
.
[2258 8589]
.
[1693 9102]
.
[2091 8700]
.
[1723 9094]
.
[2042 8739]
.
[2227 8600]
..
[2126 8682]
.
[1950 8879]
.
[2188 8586]
...
[2186 8644]
..
[2128 8679]
.
[1997 8848]
...
[1799 9061]
.
[2418 8400]
..
[2128 8683]
.
[2413 8364]
.
[2093 8669]
.
[2425 8374]
....
[2953 7833]
.
[2603 8161]
.
[2362 8381]
....
[2723 8096]
...
[2814 7955]
....
[3095 7702]
..
[2439 8346]
.
[2539 8225]
...........................
[2777 7954]
.....
[3262 7503]
.....
[3309 7450]
....
[3421 7309]
...
[4146 6559]
........
[4982 5751]
..
[6536 4221]
.
[6711 4043]
.
[4389 6322]
......
[5801 4937]
...
[4548 6140]
......
[4524 6180]
.
[6191 4546]
....
[6790 3936]
..
[5645 5078]
..................
[6142 4562]
..............................................................................................................................................................................................................................................................................................................................................
Data Dump...
.............
Data Dump...
...
Data Dump...
.
Data Dump...
.
Data Dump...

Data Dump...
...
Data Dump...
.....
Data Dump...
.......
Data Dump...
.
Data Dump...

Data Dump...
.
Data Dump...

Data Dump...
....
Data Dump...
..............
Data Dump...
..............................................................................................
[  55 4612]
......................................................................
[  44 4222]
.....
[  47 4192]
.
[  74 4227]
................
[  62 4008]
.....
[  72 3917]
............
[  52 4054]
.
[  67 4086]
..............
[  59 3999]
.......
[  72 3977]
.
[  57 4022]
....
[  69 4058]
.......................
[  72 3871]
....
[  82 3840]
..
[  73 3794]
......
[  85 3792]
....
[  83 3819]
...
[  82 3679]
.
[  97 3742]
.......
[  81 3766]
..
[  92 3807]
..
[  77 3855]
......
[  93 3838]
....
[  91 3733]
...
[  76 3781]
.....
[ 113 3737]
.
[ 112 3644]
......
[ 110 3587]
......
[ 104 3583]
..
[ 107 3636]
..
[ 127 3606]
....
[ 104 3565]
.
[ 131 3522]
...........
[ 116 3545]
.
[ 124 3520]
......
[ 130 3577]
.
[ 137 3564]
............
[ 154 3424]
.
[ 114 3575]
........................
[ 167 3408]
.................
[ 132 3486]
........
[ 262 3027]
....................................
[ 182 3307]
..
[ 230 3145]
....
[ 271 3052]
........
[ 230 3172]
.........
[ 245 3049]
......
[ 200 3262]
............
[ 228 3155]
...
[ 262 3090]
.......
[ 265 3071]
.....
[ 258 3032]
.
[ 266 3074]
...
Data Dump...
.
[ 336 2967]
.........
[ 245 3097]
..................
[ 319 2873]
...........
[ 397 2740]
.....
[ 331 2912]
......
[ 513 2556]
......
[ 310 2910]
.............
[ 403 2846]
..
[ 453 2669]
..............
[ 299 2927]
...
[ 434 2828]
.....................
[ 514 2589]
.........
[ 451 2582]
......................
[ 675 2341]
.........
[ 29 302]

Data Dump...
......................
[ 932 1977]
.
[ 614 2488]
............
[ 30 288]

Data Dump...

[ 616 2461]
.......................
[ 29 258]

Data Dump...

[ 37 254]

Data Dump...

[ 46 265]

Data Dump...
.
[1995  922]
...
[1230 1632]
...............
[1041 1859]
.
[1630 1272]
...........
[1038 1861]
....
[ 32 292]

Data Dump...
.
[1556 1319]
...........
[1004 1919]
........
[ 21 243]

Data Dump...
..
[1847 1119]
.........
[1952 1019]
............
[1775 1085]
.
[2020  914]
..........
[ 35 299]

Data Dump...
............
[ 26 268]

Data Dump...
............
[ 32 273]

Data Dump...
......
[ 18 275]

Data Dump...
.....
[ 34 224]
..............
[ 39 226]
..
Data Dump...
.............
[ 36 206]
........
Data Dump...
....
[ 39 234]
.....
Data Dump...
.
[ 30 231]
........
Data Dump...

Data Dump...
......
[ 28 282]
....
[ 43 254]
.......
Data Dump...
...........
Data Dump...
..........
[ 37 228]
..........
[ 35 217]
...
[ 35 221]
.
[ 31 249]
..
Data Dump...
..
[ 34 262]
.
[ 43 221]
...
Data Dump...
..
[ 33 251]
.......
Data Dump...
..
[ 37 209]
.
Data Dump...

Data Dump...
.......
[ 49 231]
...
[ 48 216]
......
Data Dump...
.......
Data Dump...
....
[ 34 208]
.
Data Dump...
..
[ 39 245]
...
Data Dump...
...
[ 33 241]
...
[ 33 211]
..
Data Dump...

Data Dump...
...
Data Dump...
....
[ 36 200]
.
[ 34 212]
...
Data Dump...

Data Dump...
.
[ 47 199]
.........
[ 30 195]
...
[ 30 205]
.....
[ 46 225]
.
Data Dump...

[ 50 224]
.
Data Dump...
...................
Data Dump...

Data Dump...

Data Dump...
.
Data Dump...
.......
Data Dump...
..............
[ 45 226]
........
[ 54 189]
.
[ 31 209]
...............
[ 40 147]
...
Data Dump...
.......
[ 49 178]
..........
[ 46 203]
.
[ 53 197]
.......
Data Dump...
......
Data Dump...
.....
[ 54 196]
...
[ 46 174]
.......
Data Dump...
.....
Data Dump...
..
Data Dump...
...
[ 47 201]
..
Data Dump...
.
[ 50 175]
...
[ 50 182]
...
[ 51 153]
....
[ 58 165]
..
[ 56 151]
..
Data Dump...

[ 50 171]
.............
[ 48 170]
.
Data Dump...
...
[ 66 167]
....
[ 52 170]
.
Data Dump...
....
Data Dump...
....
Data Dump...

[ 57 155]
...
[ 66 166]
..
[ 37 182]
...
Data Dump...

Data Dump...
.....
Data Dump...
..
Data Dump...
...
Data Dump...
...
[ 70 147]
...
[ 60 145]
...
[ 50 170]
.
[ 59 146]
.
Data Dump...
..
[ 68 147]
..
Data Dump...

Data Dump...
.
Data Dump...
....
Data Dump...
....
[ 67 143]
.............
[ 86 160]
..
Data Dump...

[ 67 146]
.
Data Dump...
..
Data Dump...
..
[ 77 109]
..
Data Dump...
.
[133  95]
.
Data Dump...
.
[ 91 132]
.
[90 90]
.......
[102  93]
.
[ 81 118]
...
Data Dump...

[117 105]
...
[ 74 117]
...
[135 104]
.
Data Dump...
.
Data Dump...
...
[116  99]
...
[121  97]
...
Data Dump...

Data Dump...
.
Data Dump...

Data Dump...
......
Data Dump...
.
Data Dump...

Data Dump...

Data Dump...

Data Dump...
..
Data Dump...

Data Dump...
High 2 7
Done
